mods/ra/weapons/other.yaml

```diff
@@ -186,10 +186,10 @@ TTankZap:
 DogJaw:
 	ValidTargets: Infantry
 	ReloadDelay: 10
-	Range: 3c0
+	Range: 2c256
```

Comment:
I don't really know how the original `RULES.INI` values have to be converted to match the OpenRA distances. Based it on a rough visual comparison. Also we probably want to rebalance this properly. I never saw someone on @cncnet actually train dogs.